src/System.IO.FileSystem/tests/File/ReadWriteAllBytes.cs

```diff
@@ -55,6 +55,30 @@ public void ValidWrite(int size)
         }
 
         [Fact]
+        [OuterLoop]
+        public void ReadFileOver2GB()
+        {
+            string path = GetTestFilePath();
+            using (var writer = new StreamWriter(File.Create(path)))
+            {
+                int split = 10;
+                for (int i = 0; i < (split + 1); i++)
+                {
+                    try
+                    {
+                        writer.Write(new string('a', Int32.MaxValue / split));
```

Comment:
Might be nice to create this string once outside of the loop and then just write the same string out multiple times.